GroceryStore/GroceryStore.Models/ProductClasses/Drink.cs

```diff
@@ -0,0 +1,26 @@
+﻿using GroceryStore.Constants;
+
+namespace GroceryStore.Models.ProductClasses
+{
+    public class Drink : Product
+    {
+        public bool IsAlcohol { get; set; }
+        public float Volume { get; set; }
+
+        public Drink(string name, double price, float volume, bool isAlcohol = false) :
+            base(name, price, expirationDays: 30)
+        {
+            Volume = volume;
+            IsAlcohol = isAlcohol;
+            Category = ProductCategories.Fish;
```

Comment:
I think you assigned the wrong type

Reply:
Also, according to the task the category should be passed as a parameter to the base class constructor